### ISSUE TITLE
[WinUI] pressing enter when focused on TextBox with CommandExtensions.Command attached crash the app

### ISSUE THREAD

**Issue by takla21** [unoplatform/uno.toolkit.ui#444]
## Current behavior

![image](https://user-images.githubusercontent.com/17461593/212183572-3aab598b-de32-427d-b358-0f3fa5e5bc49.png)

Error thrown
```
Exception {"Element not found. (0x80070490)"}
```

Callstack
```
CommandExtensionsCrash.Windows.dll!CommandExtensionsCrash.App.InitializeComponent.AnonymousMethod__7_1(object sender = {CommandExtensionsCrash.App}, Microsoft.UI.Xaml.UnhandledExceptionEventArgs e = {Microsoft.UI.Xaml.UnhandledExceptionEventArgs}) Line 68
	at C:\p\CommandExtensionsCrash\CommandExtensionsCrash.Windows\obj\x86\Debug\net6.0-windows10.0.19041.0\win10-x86\App.g.i.cs(68)
Microsoft.WinUI.dll!WinRT._EventSource_global__Microsoft_UI_Xaml_UnhandledExceptionEventHandler.EventState.GetEventInvoke.AnonymousMethod__1_0(object sender, Microsoft.UI.Xaml.UnhandledExceptionEventArgs e)
Microsoft.WinUI.dll!ABI.Microsoft.UI.Xaml.UnhandledExceptionEventHandler.Do_Abi_Invoke(System.IntPtr thisPtr, System.IntPtr sender, System.IntPtr e)
[Native to Managed Transition]
[Managed to Native Transition]
Microsoft.WinUI.dll!ABI.Microsoft.UI.Xaml.IApplicationStaticsMethods.Start(WinRT.IObjectReference _obj, Microsoft.UI.Xaml.ApplicationInitializationCallback callback)
Microsoft.WinUI.dll!Microsoft.UI.Xaml.Application.Start(Microsoft.UI.Xaml.ApplicationInitializationCallback callback)
CommandExtensionsCrash.Windows.dll!CommandExtensionsCrash.Program.Main(string[] args = {string[0]}) Line 31
	at C:\p\CommandExtensionsCrash\CommandExtensionsCrash.Windows\obj\x86\Debug\net6.0-windows10.0.19041.0\win10-x86\App.g.i.cs(31)
```

## Expected behavior
The command should be executed without crashing the app.

![expected](https://user-images.githubusercontent.com/17461593/212183685-85cc98da-ab44-470e-a09c-d35d232b91a6.gif)

## How to reproduce it (as minimally and precisely as possible)

- Download this sample [CommandExtensionsCrash.zip](https://github.com/unoplatform/uno.toolkit.ui/files/10406096/CommandExtensionsCrash.zip)
- Deploy on WinUI
- Make sure to focus on the textbox and press enter

## Environment

Nuget Package: Uno.Toolkit.WinUI

Package Version(s): 2.5.0-dev.8

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renders for Xamarin.Forms
- [x] Windows
- [ ] Build tasks

Visual Studio:

- [x] 2022 (version: 17.4.3)
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by Xiaoy312** [comment]
fixed in #440 already

**Comment by jeromelaban** [comment]
@Xiaoy312 do we need to backport #440 ?

**Comment by takla21** [comment]
Hum the weird part is that I tested with the latest version available **2.5.0-dev.8**. Although, the PR seems to have been merged yesterday. Maybe there are no build available with this fix yet

**Comment by takla21** [comment]
The bug is indeed fix in **2.5.0-dev.10**
